backend/models/notification.js

```diff
@@ -0,0 +1,255 @@
+/**
+ *  Copyright (C) 2018 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+"use strict";
+const hasWriteAccessToModel = require("../middlewares/checkPermissions").hasWriteAccessToModelHelper;
+const job = require("../models/job");
+const utils = require("../utils");
+const uuid = require("node-uuid");
+const db = require("../db/db");
+const _ = require("lodash");
+
+const types = {
+	ISSUE_ASSIGNED : "ISSUE_ASSIGNED",
+	ISSUE_CREATED : "ISSUE_CREATED"
+};
+
+const NOTIFICATIONS_DB = "notifications";
+const MODELS_COLL = "settings";
```

Comment:
constants.js again, and i would probably call it `MODEL_SETTINGS_COL` or something, as MODELS_COLL can get confused with the .scene, .history etc